includes/spoolchanges.js

```diff
@@ -73,7 +73,7 @@ module.exports = function(dbClient, log, bufferSize, tolerance = 600000) {
 
   const changesParams = {
     db: dbClient.dbName,
-    seqInterval: 10000
+    seqInterval: bufferSize
```

Comment:
This is now coming from outside, so potentially can be undefined. Do we maybe want to default it to an old value, similar how we do with `tolerance`?

Reply:
Good point. Since we apply a default value for `bufferSize` it will never be `undefined` from usages in the backup code. However, it is possible that test code could call it `undefined`, so I've added the same as our normal default `500` just to be safe.